notifiers/list.go

```diff
@@ -48,6 +48,10 @@ func List(ctx context.Context, app string) error {
 }
 
 func eventTypesToString(eventTypes []scalingo.EventType, ids []string) (res string) {
+	if len(ids) == 0 {
+		res = ""
+		return
```

Comment:
It is not related to your code but could you please replace the "naked" returns by return with the variable name as it is not correspond to our style guide.
It is usually appreciated to improve the legacy code when you can, if you have time to do it :)